src/inputs/internal/ComboBoxInput.tsx

```diff
@@ -92,23 +92,28 @@ export function ComboBoxInput<O, V extends Value>(props: ComboBoxInputProps<O, V
   const multilineProps = allowWrap ? { textAreaMinHeight: 0, multiline: true } : {};
   useGrowingTextField({ disabled: !allowWrap, inputRef, inputWrapRef, value: inputProps.value });
 
+  const chipLabels = isTree ? selectedOptionsLabels || [] : selectedOptions.map((o) => getOptionLabel(o));
+
   return (
     <TextFieldBase
       {...otherProps}
       {...multilineProps}
+      unfocusedPlaceholder={showNumSelection && <Chips compact={otherProps.compact} values={chipLabels} />}
```

Comment:
**question** Is this going to be displaying Chips for single select fields?

Reply:
Nope. `showNumSelection` specifes `> 1`

Reply:
circling back to this all select fields will show a chip now